examples/stable/postgresql-ha/hook-blueprint/README.md

```diff
@@ -0,0 +1,171 @@
+
+## Introduction
+
+This Helm chart has been developed based on [Postgresql](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) chart but including some changes to guarantee high availability such as:
+
+- A new deployment, service have been added to deploy [Pgpool-II](https://pgpool.net/mediawiki/index.php/Main_Page) to act as proxy for PostgreSQL backend. It helps to reduce connection overhead, acts as a load balancer for PostgreSQL, and ensures database node failover.
+- Replacing `bitnami/postgresql` with `bitnami/postgresql-repmgr` which includes and configures repmgr. Repmgr ensures standby nodes assume the primary role when the primary node is unhealthy.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters. This chart has been tested to work with NGINX Ingress, cert-manager, fluentd and Prometheus on top of the [BKPR](https://kubeprod.io/).
+
+## Requirements
+When restoring the postgres with high availability in a different namespace, the standby instance pod goes into ``CrashLoopBackOff`` since the connection info for the primary/secondary nodes in the ``repmgr`` database points to source namespace. The blueprint ``postgres-ha-hook.yaml`` can be used to solve this issue which will update the ``repmgr`` database with correct connection information for primary and secondary instances
+
+## Prerequisites
+
+- Kubernetes 1.10+
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.71.0 installed in your cluster
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+
+## Installing the Chart
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
+
+# If you are using helm v3
+$ kubectl create ns postgres-ha-test
+$ helm install my-release --namespace postgres-ha-test bitnami/postgresql-ha
+
+# If you are using helm v2
+$ helm install --name my-release --namespace postgres-ha-test bitnami/postgresql-ha
+```
+
+The command deploys PostgreSQL HA on the Kubernetes cluster in the default configuration.
+
+> **Tip**: List all releases using `helm list`
+
+In case, if you don't have `Kanister` installed already, you can use following commands to do that.
+Add Kanister Helm repository and install Kanister operator
+```bash
+$ helm repo add kanister https://charts.kanister.io
+$ helm install --name kanister --namespace kasten-io kanister/kanister-operator --set image.tag=0.71.0
+
+```
+
+### Create Blueprint
+Create Blueprint in the same namespace as the controller
+
+```bash
+$ kubectl create -f ./postgres-ha-hook-blueprint.yaml -n kasten-io
+```
+
+### After Restore of the application
+Lets assume, restore is done in a namespace `postgres-ha-test2`
+
+The pod ``my-release-postgresql-ha-postgresql-1`` in ``postgres-ha-test2`` goes into ``CrashLoopBackoff`` because of wrong connection information
```

Comment:
```suggestion
The pod ``my-release-postgresql-ha-postgresql-1`` in ``postgres-ha-test2`` goes into ``CrashLoopBackoff`` because of wrong connection information.
```